src/Symfony/Component/Form/Tests/AbstractLayoutTest.php

```diff
@@ -71,17 +71,15 @@ protected function assertMatchesXpath($html, $expression, $count = 1)
             ));
         }
         $xpath = new \DOMXPath($dom);
-        $nodeList = $xpath->evaluate('/root'.$expression);
+        $nodeList = $xpath->evaluate('/html/body'.$expression);
 
         if ($nodeList->length != $count) {
-            $dom->formatOutput = true;
             $this->fail(sprintf(
                 "Failed asserting that \n\n%s\n\nmatches exactly %s. Matches %s in \n\n%s",
                 $expression,
                 1 == $count ? 'once' : $count.' times',
                 1 == $nodeList->length ? 'once' : $nodeList->length.' times',
-                // strip away <root> and </root>
-                substr($dom->saveHTML(), 6, -8)
+                $html
```

Comment:
formatted output not worth the hassle IMHO. As saveHTML includes a doctype declaration _either way_.